.github/workflows/main.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
   test-build-and-push:
     name: Test, build and push
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master'
+    if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/CleanupMineFilter'
```

Comment:
Dette kan kanskje fjernes når før det går til dev?

Reply:
yep, I agree...it was just temporary solution to test changes.